### PR TITLE
feat(cli): support specify port and host

### LINKED ISSUE #92: auto get available port in dev
**Is your feature request related to a problem? Please describe.**
image

Currently, if port 1234 is occupied, an error is thrown directly, but there is no way to change the port.

**Describe the solution you'd like**

* There should be a way to specify the port when dev, for example `lagon dev --port 8000`
* Ability to automatically get available port([get-port-please](https://www.npmjs.com/package/get-port-please))

### PR DESCRIPTION
**Pull request by await-ovo**
Closes #92

### CODE REVIEW

File: packages/cli/src/cli.ts [hunk 1]
```diff
@@ -28,6 +28,8 @@ export function runCli() {
     .command('dev')
     .description('Launch a local dev server')
     .argument('<file>', 'The file to serve')
+    .option('--port [port]', 'Specify dev server port number', '1234')
```
Comment (QuiiBz) [inline packages/cli/src/cli.ts:31]: Any reason why the port is optional? I don't think that should be the case, like with `--host`

File: packages/cli/src/commands/dev.ts [hunk 2]
```diff
@@ -142,15 +145,28 @@ export async function dev(file: string, { preact, publicDir }: { preact: boolean
...
+      }
     }
+  }
 
-    console.clear();
-    logSpace();
-    logSuccess(`Dev server running at: ${address}`);
-    logSpace();
-  });
+  start(Number(port));
```
Comment (QuiiBz) [inline packages/cli/src/commands/dev.ts:171]: I think we should check if `port` is a number before, to prevent any issues.

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks for this PR!

File: packages/cli/src/cli.ts (hunk 1 above)
Comment (await-ovo) [reply packages/cli/src/cli.ts:31]: Thanks your feedback， My mistake, I'll change it.

File: packages/cli/src/commands/dev.ts (hunk 2 above)
Comment (await-ovo) [reply packages/cli/src/commands/dev.ts]: you are right, i will use `parseInt` to parse port string to number.

**Review by QuiiBz — APPROVED** [review]
LGTM!

**Pull request merged**